exchanges/coinbasepro/coinbasepro_websocket.go

```diff
@@ -400,7 +399,7 @@ func (c *CoinbasePro) Subscribe(subs subscription.List) error {
 	for i, s := range subs {
 		if i == 0 {
 			r.ProductIDs = s.Pairs.Strings()
-		} else if !slices.Equal(subs[0].Pairs, s.Pairs) {
+		} else if subs[0].Pairs.Equal(s.Pairs) {
```

Comment:
Can you clarify your first sentence for me?
We don't only check the first two. We test until we find one that doesn't match the first one.

However, this is obviously broken, because I dropped the `!` accidentally.

Fixed 9f11224

Reply:
I meant that in the broken version I reviewed, it only checked the first two entries before calling `break`. Now I know its meant to do the opposite, but your update is good, thanks! 👍